### ISSUE TITLE
SDWebImage v4 can not import libwebp framework's header files.

### ISSUE THREAD

**Issue by Whirlwind** [rs/SDWebImage#1887]
I build the libwebp source files to a static framework, so I get the framework:

```

.
└── libwebp.framework
    ├── Headers -> Versions/Current/Headers
    ├── Versions
    │   ├── A
    │   │   ├── Headers
    │   │   │   ├── alphai.h
    │   │   │   ├── backward_references.h
    │   │   │   ├── bit_reader.h
    │   │   │   ├── bit_reader_inl.h
    │   │   │   ├── bit_writer.h
    │   │   │   ├── color_cache.h
    │   │   │   ├── common.h
    │   │   │   ├── common_sse2.h
    │   │   │   ├── cost.h
    │   │   │   ├── decode_vp8.h
    │   │   │   ├── delta_palettization.h
    │   │   │   ├── dsp.h
    │   │   │   ├── endian_inl.h
    │   │   │   ├── filters.h
    │   │   │   ├── histogram.h
    │   │   │   ├── huffman.h
    │   │   │   ├── huffman_encode.h
    │   │   │   ├── lossless.h
    │   │   │   ├── mips_macro.h
    │   │   │   ├── msa_macro.h
    │   │   │   ├── muxi.h
    │   │   │   ├── neon.h
    │   │   │   ├── quant_levels.h
    │   │   │   ├── quant_levels_dec.h
    │   │   │   ├── random.h
    │   │   │   ├── rescaler.h
    │   │   │   ├── thread.h
    │   │   │   ├── utils.h
    │   │   │   ├── vp8enci.h
    │   │   │   ├── vp8i.h
    │   │   │   ├── vp8li.h
    │   │   │   ├── webp
    │   │   │   │   ├── decode.h
    │   │   │   │   ├── demux.h
    │   │   │   │   ├── encode.h
    │   │   │   │   ├── extras.h
    │   │   │   │   ├── format_constants.h
    │   │   │   │   ├── mux.h
    │   │   │   │   ├── mux_types.h
    │   │   │   │   └── types.h
    │   │   │   ├── webpi.h
    │   │   │   └── yuv.h
    │   │   └── libwebp
    │   └── Current -> A
    └── libwebp -> Versions/Current/libwebp
```

The SDWebImage will failed to ` #import "webp/decode.h"`.

`#import "webp/decode.h"` will only work for the source file version.

I hope that we can change the import way to `#import <libwebp/webp/decode.h>`, it will work for framework version and source file version.

**Comment by Whirlwind** [comment]
The xcconfig in SDWebImage's podspec is unnecessary:

```
          "USER_HEADER_SEARCH_PATHS": "$(inherited) $(SRCROOT)/libwebp/src"
```

**Comment by Whirlwind** [comment]
@rs Could you have a look at this issue?

**Comment by dreampiggy** [comment]
Maybe this is what you prefer? #1983

**Comment by bpoplauschi** [comment]
Fixed by #1983
